benchmarks/cugraph/standalone/bulk_sampling/cugraph_bulk_sampling.py

```diff
@@ -344,7 +344,7 @@ def generate_rmat_dataset(
     del label_df
     gc.collect()
 
-    dask_label_df = dask_cudf.from_dask_dataframe(dask_label_df)
+    dask_label_df = dask_label_df.to_backend("cudf")
```

Comment:
`from_dask_dataframe` is now deprecated.

Reply:
Does this mean that dask-expr has some dispatching/plugin support for different DataFrame implementations?

Reply:
Correct - Dask documentation is here: https://docs.dask.org/en/latest/how-to/selecting-the-collection-backend.html, and Dask-cudf: https://docs.rapids.ai/api/dask-cudf/stable/#dataframe-creation-from-in-memory-formats